test/E2ETest/WebStack.QA.Test.OData/ODataOrderByTest/OrderByEdmModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Data.Entity;
+using System.Web.OData.Builder;
+using Microsoft.OData.Edm;
+
+namespace WebStack.QA.Test.OData.ODataOrderByTest
+{
+    public class OrderByEdmModel
+    {
+        public class OrderByContext: DbContext
+        {
+            private static readonly string ConnectionString =
+                @"Data Source=(LocalDb)\v11.0;Integrated Security=True;Initial Catalog=OrderByTest";
+
+            public OrderByContext()
+                : base(ConnectionString)
+            {
+            }
+
+            public IDbSet<Item> Items { get; set; }
+        }
+        
+
+        public static IEdmModel GetModel()
+        {
+            ODataConventionModelBuilder builder = new ODataConventionModelBuilder();
+            builder.EntitySet<Item>("Items");
+
```

Comment:
please add a second model with different ordering to ensure it works with more than one model.